app/providers.tsx

```diff
@@ -0,0 +1,11 @@
+"use client";
```

Comment:
I'd put this under `components/layout` and capitalize it since it a react component:

`components/layout/Providers.tsx`

Reply:
Just capitalized and moved to the components folder but don't currently have a layout subfolder. Should I create one?

Reply:
yeah